ios emulation-based analysis (#1553)
OWASP/owasp-mstg: Document/0x04c-Tampering-and-Reverse-Engineering.md

#### Emulation-based Dynamic Analysis

> Emulation is an imitation of a certain computer platform or program on another platform or program. The software or hardware performing this imitation is called an _emulator_. Emulator provides a much cheaper alternatives to an actual device, where an user can manipulate it without worrying about damaging the device. There are multiple emulators available for Android, but unlike Android, for iOS there are practically no viable emulators available.

Reviewer: I've enhanced the first sentence. Does it still express the idea as intended?

```suggestion
Emulation is an imitation of a certain computer platform or program being executed in different platform or within another program. The software or hardware performing this imitation is called an _emulator_. Emulators provide a much cheaper alternative to an actual device, where a user can manipulate it without worrying about damaging the device. There are multiple emulators available for Android, but for iOS there are practically no viable emulators available.
```
